src/docs/product/alerts/alert-types.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Alert Types
+sidebar_order: 10
+redirect_from:
+  - /workflow/alerts-notifications/alerts/
+  - /workflow/notifications/alerts/
+  - /product/alerts-notifications/alerts/
+  - /product/alerts-notifications/metric-alerts/
+  - /product/alerts-notifications/issue-alerts/
+description: "Learn how to configure and manage Metric Alerts."
+---
+
+You can create two types of alerts:
+
+1. **Issue alerts**: Trigger when an issue (a grouped set of error events) matches a specific criteria.
+2. **Metric alerts**: Monitor macro-level metrics for both error events and transaction events.
```

Comment:
@adhiraj Fiona suggested we make these definitions parallel in structure. I've put in a suggestion below. Does it work for you?

```suggestion
2. **Metric alerts**: Trigger when macro-level metrics for error or transaction events cross specific thresholds.
```